### DISCUSSION TITLE
AirByte to load data (SQLSERVER) into StarRocks

### DISCUSSION THREAD

**Discussion by sejubar** [airbytehq/airbyte#33701] [Connector Questions]
I am testing out Syncing two tables from sqlserver to starrocks using the connector `starrocks/destination-starrocks:latest` 
But I am getting the errors below;

`io.airbyte.integrations.destination.starrocks.exception.StreamLoadFailException: Stream load failed because of unknown exception, db: POG, table: _airbyte_tmp_zxg_Order, label: airbyte__airbyte_tmp_zxg_Order_597527e7-704c-4693-8dfe-8004d4e355e41703110794930`

`
 {
  "failureOrigin" : "replication",
  "internalMessage" : "Destination process is still alive, cannot retrieve exit value.",
  "externalMessage" : "Something went wrong during replication",
  "metadata" : {
    "attemptNumber" : 1,
    "jobId" : 6
  },
`

any help will be appreciated.

**Comment by marcosmarxm** [comment]
This connector is not supported in Airbyte core project. Kindly ask you to request to StarRocks to help you.
